platforms/azure/variables.tf

```diff
@@ -164,3 +164,36 @@ EOF
 
   default = ""
 }
+
+variable "tectonic_azure_byo_etcd_nsg" {
```

Comment:
Can we use 'external' instead of 'byo' as we're already standardizing on that terminology? e.g. https://github.com/coreos/tectonic-installer/blob/master/examples/terraform.tfvars.azure#L26